README.md

```diff
@@ -589,6 +589,61 @@ ort {
 }
 ```
 
+## Experimental Scanner
+
+ORT provides an alternative scanner implementation which is currently called "experimental scanner". By now, this
+implementation can be considered stable, and it will replace the default scanner implementation in the middle of
+September 2022. Therefore, we encourage all ORT users to test the new implementation and report any discovered issues.
+
+The main difference to the old implementation is that the experimental scanner groups packages by their provenance
+before scanning. This ensures that a certain revision of a VCS repository is only scanned once, and the results are
+shared for all packages that are provided by this repository. In the case of repositories that provide a lot of
+packages, this can bring a significant performance improvement.
+
+Also, the new implementation better tracks if the VCS revision provided by the metadata of a package points to a moving
+revision, like a Git branch, and resolves the revision before reusing any scan results from a storage. This fixes an
+issue where ORT would reuse scan results for a branch even if they do not match the current revision of the branch
+anymore (see https://github.com/oss-review-toolkit/ort/issues/4562).
```

Comment:
Nit: Should we link text instead of just pasting the issue URL?

Reply:
GitHub replaces the URL with the issues ID so I think that is fine: https://github.com/oss-review-toolkit/ort/blob/953bfa4e6720125a11b46fed02b05d4a74aca415/README.md#experimental-scanner